pandas/tests/frame/methods/test_select_dtypes.py

```diff
@@ -88,6 +88,29 @@ def test_select_dtypes_exclude_include_using_list_like(self):
         e = df[["b", "e"]]
         tm.assert_frame_equal(r, e)
 
+    def test_select_dtypes_exclude_include_int(self):
+        df = DataFrame(
+            {
+                "a": list("abc"),
+                "b": list(range(1, 4)),
+                "c": np.arange(3, 6, dtype="int32"),
+                "d": np.arange(4.0, 7.0, dtype="float64"),
+                "e": [True, False, True],
+                "f": pd.date_range("now", periods=3).values,
+            }
+        )
+        exclude = (np.datetime64,)
+        include = np.bool_, "int"
+        r = df.select_dtypes(include=include, exclude=exclude)
```

Comment:
can you also add the case where inlclude='integer'

Reply:
Ok